test/units/modules/network/f5/test_bigip_snat_pool.py

```diff
@@ -99,6 +101,7 @@ def test_create_snat_pool(self, *args):
             name='my-snat-pool',
             state='present',
             members=['10.10.10.10', '20.20.20.20'],
+            description='A SNAT pool description',
```

Comment:
this is not needed if you want you can create a separate unit test to test create with description

Reply:
Is it enough to keep it in the test_module_parameters function?